cvrf2csaf/config/config.yaml

```diff
@@ -6,9 +6,14 @@ cvrf2csaf_name: CVRF-CSAF-Converter
 # Force conversion, produces invalid output to be fixed manually
 force: no
 
+# Document (top-level section)
+csaf_version: TBD-IN-CONFIG
```

Comment:
This should default to `2.0` (as a string).

Reply:
> In general, I agree on that change. However, the class name is confusing. It should mention that it cares only about those in `/document`.

Ok, we will document the CSAF path which the classes are handling.

What class name would you prefer? DocumentLeafElements?

Reply:
> > In general, I agree on that change. However, the class name is confusing. It should mention that it cares only about those in `/document`.
> 
> Ok, we will document the CSAF paths which the classes are handling.
> 
> What class name would you prefer? DocumentLeafElements?

I don't have a better suggestion right now, so I could live with `DocumentLeafElements` instead of `ToplevelLeafElements`.

@sthagen: Do you have a better name?

Reply:
@tschmidtb51 Maybe `class HodgePodge` (the class even mixes in the CSAF version element from the future ...)?
 I read the #47 ingredients list again and I think `DocumentLeafElements` is good 'nuff. :+1:

Reply:
@sustefil `DocumentLeafElements` it is.